frontend/src/scenes/surveys/surveyViewViz.tsx

```diff
@@ -686,7 +736,7 @@ function ResponseSummaryFeedback({ surveyId }: { surveyId: string }): JSX.Elemen
             return // Already rated
         }
         setRating(newRating)
-        posthog.capture('chat rating', {
+        posthog.capture('survey_resonse_rated', {
```

Comment:
Do we use the event's name in insights or something that would break otherwise?

Reply:
it's not been captured before... it was copy pasta from the existing max chat component, but won't overlap with any existing events 👍